lib/main.coffee

```diff
@@ -198,6 +198,7 @@ module.exports =
         if text.indexOf("\n") is -1
           imageToken = if isImage then '!' else ''
           if text.match(/[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/)
+            if text.match(/\.(jpg|jpeg|png|gif|tiff)/) then imageToken = '!'
```

Comment:
Definitely; not many though. The thing is that you're making an assumption that might not be true. I'm not entirely sure though. I think the number of use cases is limited, but I don't like making assumptions. Could you comment it out for now? We can always enable it again.